docs/reference/commands/node-tool.asciidoc

```diff
@@ -0,0 +1,335 @@
+`elasticsearch-node` tool to construct a new cluster that contains some of the
+data from the failed cluster.
+
+This tool has two modes:
+
+* `elastisearch-node unsafe-bootstap` can be used if there is at least one
+  remaining master-eligible node. It forces one of the remaining nodes to form
+  a brand-new cluster on its own, using its local copy of the cluster metadata.
+  This is known as _unsafe cluster bootstrapping_.
+
+* `elastisearch-node detach-cluster` allows you to move nodes from one cluster
+  to another. This can be used to move nodes into the new cluster created with
+  the `elastisearch-node unsafe-bootstap` command above. It also allows you to
+  move nodes into a brand-new cluster if unsafe cluster bootstrapping was not
+  possible.
+
+[[node-tool-unsafe-bootstrap]]
+[float]
+==== Unsafe cluster bootstrapping
+
+If there is at least one remaining master-eligible node, but it is not possible
+to restart a majority of them, then the `elasticsearch-node unsafe-bootstrap`
+command will unsafely override the cluster's <<modules-discovery-voting,voting
+configuration>> as if performing another
+<<modules-discovery-bootstrap-cluster,cluster bootstrapping process>>, allowing
+the target node to form a new cluster on its own. This new cluster forms using
+the cluster metadata held locally on the target node. This can lead to
+arbitrary data loss since the target node may not hold the latest cluster
+metadata, and this out-of-date metadata may make it impossible to use some or
+all of the indices in the cluster.
+
+Since unsafe bootstrapping forms a new cluster containing a single node, once
+you have run it you must use the <<node-tool-detach-cluster,`elasticsearch-node
+detach-cluster` tool>> to migrate any other surviving nodes from the failed
+cluster into this new cluster.
+
+When you run the `elasticsearch-node unsafe-bootstrap` tool it will analyse the
+state of the node and ask for confirmation before taking any action. Before
+asking for confirmation it reports the term and version of the cluster state on
+the node on which it runs as follows:
+
+[source,txt]
+----
+Current node cluster state (term, version) pair is (4, 12)
+----
+
+If you have a choice of nodes on which to run this tool then you should choose
+one with a term that is as large as possible, and if there is more than one
+node with the same term then you should pick the one with the largest version.
+This identifies the node with the freshest cluster state, minimising the
+quantity of data that might be lost. For example, if the first node reports
+`(4, 12)` and a second node reports `(5, 3)`, then the second node is preferred
+since its term is larger.  However if the second node reports `(3, 17)` then
+the first node is preferred since its term is larger. If the second node
+reports `(4, 10)` then it has the same term as the first node, but has a
+smaller version, so the first node is preferred.
+
+[WARNING]
```

Comment:
I suggested turning an earlier mention of data loss into a warning, so you can either leave both or combine this with the earlier one.